src/coord/config.c

```diff
@@ -267,5 +307,25 @@ int RegisterClusterModuleConfig(RedisModuleCtx *ctx) {
     )
   )
 
+  if (clusterConfig.type == ClusterType_RedisOSS) {
+    if (RedisModule_RegisterStringConfig (
+          ctx, "search-oss-global-password", "",
+          REDISMODULE_CONFIG_IMMUTABLE | REDISMODULE_CONFIG_UNPREFIXED,
```

Comment:
Should we use `REDISMODULE_CONFIG_SENSITIVE` flag as well to redact it in logging?

Reply:
Yes. Flag added